### PR TITLE
[FSDE-1240] Added Chi Vue section to installation page

### PR DESCRIPTION
**Pull request by janibolkvadze**

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-116/1/. :white_check_mark:

File: src/website/views/getting-started/installation.pug [hunk 1]
```diff
@@ -178,7 +178,97 @@ p.-text.-mb--3
...
+  | Chi supports a set of native Vue components.
+  | Chi Vue can be installed via Github npm registry or included from CDN as a UMD library.
+ul#chi-vue-tabs.chi-tabs.-border
+  li.-active
+    a(href='#chi-vue-tabs--npm') NPM
+  li
+    a(href='#chi-vue-tabs--umd') UMD
+.-no-example-code-snippet
+  #chi-vue-tabs--npm.chi-tabs-panel.-active
+    .-d--flex.-no-gutter.-bt--1.-bg--grey-15
```
Comment (jllr) [inline src/website/views/getting-started/installation.pug:194]:
```suggestion
    .-d--flex.-no-gutter.-bg--grey-15
```

**Review by jllr — APPROVED** [review]

**Review by aryaroudi — APPROVED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-116/2/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-116/3/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-116/4/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://08a04a44374b8c555e3b3a6ef693faa04bc868ed-chi-pr.enterprise.ctl.io

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-116/5/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline did not run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-116/6/. :x:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-116/7/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://be31aa4ea46b222914b6e7bd34552913a440ca2f-chi-pr.enterprise.ctl.io

**Pull request merged**